snapcraft.yaml.template

```diff
@@ -0,0 +1,75 @@
+name: ECLIPSE_PACKAGE
+base: core22
+
+version: 'ECLIPSE_VERSION'
+summary: Extensible Tool Platform and Java IDE
+description:
+  Eclipse provides IDEs and platforms for nearly every language and architecture.
+  We are famous for our Java IDE, C/C++, JavaScript and PHP IDEs built on extensible
+  platforms for creating desktop, Web and cloud IDEs. These platforms deliver the most
+  extensive collection of add-on tools available for software developers.
+
+grade: stable
+confinement: strict
```

Comment:
Unfortunately, we can't create IDEs that have strict confinement.

Since IDE's need to access files anywhere on the system, and integrate with build tools installed on the host system, they should only run in classic confinement. Although compiling simple examples should work, more thorough examples will cause issues.

This means you won't be able to use the `gnome` extension, unfortunately.